wrapper/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func addProcessToCgroup(fn string, pid int) {
+	file, err := os.OpenFile(fn, os.O_WRONLY, 0644)
+	if err != nil {
+		log.Println("unable to open", fn, "for writing :", err)
+		os.Exit(1)
+	}
+	defer file.Close()
+
+	if _, err := file.WriteString(fmt.Sprintf("%d", pid)); err != nil {
+		log.Println("failed to add pid to cgroup: ", err)
+		os.Exit(1)
+	}
+}
+
+func main() {
+	// wrapper procfile cmd args...
+	if len(os.Args) < 3 {
+		log.Fatalln("too few args to wrapper")
+		os.Exit(1)
```

Comment:
`log.Fatalln` already calls `os.Exit(1)`:
```suggestion
		log.Fatalln("too few args to wrapper")
```

This also applies to the other calls below.